app/views/trips/_import_form.html.haml

```diff
@@ -0,0 +1,6 @@
+=form_for @trip, action: "import", method: "post", url: "" do |f|
```

Comment:
This form isn't reused anywhere else for now, it could be easier if you just put it in the regular `import.html.haml` view. Up to you though